examples/gfx/opengl/3_opengl_3d_cube/main.cpp

```diff
@@ -94,12 +94,12 @@ int main() {
 
     f32 t = 0.0f;
     while (!window->closing()) {
-        ++t;
+        t += Time::get_delta_time();
 
         GLContext::clear_color(0.2f, 0.2f, 0.2f, 1.0f);
         GLContext::clear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
 
-        transform_ptr->rotation = trait_bryan_angle_yxz(Vec3f(t * 0.01f, t * 0.01f, 0.0f));
+        transform_ptr->rotation = trait_bryan_angle_yxz(Vec3f(t, t, 0.0f));

```

Comment:
Hey, maybe for convenience, lets add Time::running_time(), akka application running time.

![image](https://github.com/Maksasj/bebone/assets/36482401/9beda3de-f838-4327-8863-d8e98ee8df0e)

as I see we have application startup time, so what will not be diffucult

Reply:
Yeah, that would be great

Reply:
@Maksasj I created [Time::get_seconds_elapsed](https://github.com/Maksasj/bebone/blob/7a746e9304ecb445e811743ed160df386a8e3253/src/bebone/core/timestamp/timestamp.cpp#L19-L22). Maybe naming is not the best, but this method retrevies application running time

Reply:
> @Maksasj I created [Time::get_seconds_elapsed](https://github.com/Maksasj/bebone/blob/7a746e9304ecb445e811743ed160df386a8e3253/src/bebone/core/timestamp/timestamp.cpp#L19-L22). Maybe naming is not the best, but this method retrevies application running time

nice !